package.json

```diff
@@ -7,39 +7,44 @@
   "author": "jbcpollak <josh@6river.com> (http://6river.com/)",
   "main": "server/server.js",
   "engines": {
-    "node": "5.7.1",
+    "node": "6.2.1",
     "npm": "3.7.2"
   },
   "scripts": {
-    "debug": "NODE_ENV=development node debug .",
-    "start": "NODE_ENV=development nodemon .",
+    "debug": "NODE_ENV=development NODE_PATH=$NODE_PATH:./ node debug .",
+    "start": "NODE_ENV=development NODE_PATH=$NODE_PATH:./ nodemon .",
     "pretest": "eslint .",
-    "test": "NODE_ENV=test istanbul cover node_modules/.bin/_mocha",
-    "test:no-cover": "NODE_ENV=test mocha",
+    "test": "NODE_ENV=test NODE_PATH=$NODE_PATH:./ istanbul cover node_modules/.bin/_mocha -- ${MOCHA_OPTS}",
+    "test:no-cover": "NODE_ENV=test NODE_PATH=$NODE_PATH:./ mocha",
     "test:debug": "npm run test:no-cover -- --debug-brk",
     "posttest": "nsp check"
   },
   "dependencies": {
+    "chai-as-promised": "^5.3.0",
     "compression": "=1.6.2",
     "cors": "=2.7.1",
+    "es6-promisify": "^4.1.0",
     "helmet": "=2.1.0",
-    "loopback": "=2.28.0",
-    "loopback-boot": "=2.18.1",
-    "loopback-component-explorer": "=2.5.0",
-		"loopback-component-bunyan": "git://github.com/6RiverSystems/loopback-component-bunyan.git#v0.0.3",
-		"loopback-datasource-juggler": "=2.46.0",
-    "serve-favicon": "=2.3.0"
+    "loopback": "^2.29.1",
+    "loopback-boot": "^2.20.0",
+		"loopback-component-bunyan": "git://github.com/6RiverSystems/loopback-component-bunyan.git#5917459",
```

Comment:
I would prefer leaving this at a release version I think, but I dont feel too strongly about it - I consider the current commits experimental